### PR TITLE
Minigame victory overhaul

### PR DESCRIPTION
**Pull request by ThomasMartinOctoPi**
Animation is fixed, gamegrid is now 3x3, unused Tubes are removed after victory, player can now return to Overworld by pressing the screen after Victory

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -13,9 +14,12 @@ export enum Scene {
 export type SceneProps = {
 	setSceneHook: (newActiveScene: Scene) => void;
 	visible?: boolean;
+	setplayerpos?: (setplayerpos: Vector3) => void;
```
Comment (timowenz) [inline src/App.tsx:17]: setPlayerPos

File: src/App.tsx [hunk 2]
```diff
@@ -13,9 +14,12 @@ export enum Scene {
...
+	playerpos?: Vector3;
```
Comment (timowenz) [inline src/App.tsx:18]: playerPos

File: src/App.tsx [hunk 3]
```diff
@@ -13,9 +14,12 @@ export enum Scene {
...
 };
 
 export default function App() {
+	const [playerstartingpos, setplayerstartingpos] = useState<Vector3>(new Vector3(0, -0.3, 0));
```
Comment (timowenz) [inline src/App.tsx:22]: playerStartingPos, setPlayerStartingPos

**Review by timowenz — CHANGES_REQUESTED** [review]
Camel case to every variable

**Review by timowenz — APPROVED** [review]

**Comment by ThomasMartinOctoPi** [comment]
Inshallah

**Pull request merged**